A72-open-telemetry-tracing.md

```diff
@@ -0,0 +1,554 @@
+A72: OpenTelemetry Tracing 
+----
+* Author(s): [Yifei Zhuang](https://github.com/YifeiZhuang), [Yash Tibrewal](https://github.com/yashykt)
+* Approver: [Eric Anderson](https://github.com/ejona86)
+* Reviewers: [Mark Roth](https://github.com/markdroth), [Doug Fawley](https://github.com/dfawley),
+[Feng Li](https://github.com/fengli79)
+* Status: Ready for Implementation
+* Implemented in: <language, ...>
+* Last updated: 2024-01
+* Discussion at: https://groups.google.com/g/grpc-io/c/e_ByaRmtJak
+
+## Abstract
+This proposal adds support for OpenTelemetry tracing and suggests migration
+paths away from OpenCensus tracing. Discussions include:
+* Context propagation between a gRPC client and server during migration.
+* Migrate a gRPC involved software binary that has both OpenTelemetry and 
+OpenCensus dependency.
+* The API surface to enable and configure OpenTelemetry tracing.
```

Comment:
General comment about the organization of this doc:

The doc seems to be currently organized to present OT tracing primarily in the context of OC tracing.  However, this is likely to be confusing to new users that are not already using OC and just want to understand how OT tracing works.  And in the long reun, I think this doc will be read by more users like than than by users who are already using OC.

Given that, I think it would make more sense to first describe the OT tracing functionality from scratch, without referring to OC.  Then after that, we can have a section describing migration from OC, which will include things like the differences in trace-span annotations, `grpc-trace-bin` propagation, migration use-cases, etc.

Reply:
Hm, describing OT first followed by the migration stuff in later section seems indeed very clear. Re-arranged. Thanks for the suggestion.